airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -995,6 +995,8 @@ def create_batch(
         """
```

Comment:
From my point of view, all these messages should be displayed on the operators level, so maybe we can move them from hook?

Reply:
@VladaZakharova other logs are already present at operator level so just added one more.

Reply:
I think it should be in the hook level because some other operators can use the same hook and better the log be in one place (hook) instead of each operator.